src/textual/validation.py

```diff
@@ -419,12 +423,19 @@ def describe_failure(self, failure: Failure) -> str | None:
             A string description of the failure.
         """
         if isinstance(failure, Length.Incorrect):
+            desc: str = ""
             if self.minimum is None and self.maximum is not None:
-                return f"Must be shorter than {self.maximum} characters."
+                desc += f"Must be shorter than {self.maximum} characters."
             elif self.minimum is not None and self.maximum is None:
-                return f"Must be longer than {self.minimum} characters."
+                desc += f"Must be longer than {self.minimum} characters."
             else:
-                return f"Must be between {self.minimum} and {self.maximum} characters."
+                desc += f"Must be between {self.minimum} and {self.maximum} characters."
+
+            desc += (
```

Comment:
I think we could probably do without this messages. For non-engineers, you wouldn't typically count spaces anyway.